Sprint-2/implement/contains.test.js

```diff
@@ -20,16 +20,39 @@ as the object doesn't contains a key of 'c'
 // Given an empty object
 // When passed to contains
 // Then it should return false
-test.todo("contains on empty object returns false");
+test("contains on empty object returns false", () => {
+  const inputEmpty = [];
```

Comment:
This isn't an empty object, it's an array. It may be interesting to test what happens when passed an array, but it's not quite what the test name describes.